src/api/services/candidates.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import { Candidate } from "../../models/Candidate";
+
+const CANDIDATE_API_URL = "http://localhost:3000/candidates";
```

Comment:
You should move this to env file and import as baseURL all over the app. http://localhost:3000